aws/resource_aws_workspaces_directory.go

```diff
@@ -140,6 +140,8 @@ func resourceAwsWorkspacesDirectoryCreate(d *schema.ResourceData, meta interface
 
 func resourceAwsWorkspacesDirectoryRead(d *schema.ResourceData, meta interface{}) error {
 	conn := meta.(*AWSClient).workspacesconn
+	ignoreTags := meta.(*AWSClient).ignoreTags
```

Comment:
Hi @ewbankkit 👋 We would prefer to not add resources/logic to the `ignore_tags` preview at this time. We are finishing up a round of feedback on the functionality, so the implementation may change.

Reply:
@bflad I was using the `ignore_tags` functionality to test the claim of the [`CreateTags`](https://docs.aws.amazon.com/workspaces/latest/api/API_CreateTags.html) API:

> If you want to add new tags to a set of existing tags, you must submit all of the existing tags along with the new ones.

by having tags that are not managed by Terraform and seeing if they get removed when updating tags via Terraform - it turns out they don't so the API in fact doesn't overwrite any existing tags.
I'll change this PR to _WIP_ and remove the `ignore_tags` stuff.
I'll use the same approach to test the S3 object tagging functionality in https://github.com/terraform-providers/terraform-provider-aws/pull/11964 but again not leave the changes in the final PR.